socorro/unittest/external/es/base.py

```diff
@@ -301,11 +311,10 @@
         'permissions_needed': [],
         'query_type': 'string',
         'storage_mapping': {
-            'type': 'string',
+            'type': 'text',
             'fields': {
                 'full': {
-                    'type': 'string',
-                    'index': 'not_analyzed',
+                    'type': 'keyword',
```

Comment:
Also, all of these are only ever used for unit tests. The actual mapping that we use in production are inside our database, in the Super Search Fields list. See https://crash-stats.mozilla.com/admin/supersearch-fields/